packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -5,22 +5,43 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import {
```

Comment:
As a note for the future to tack on to @xyc's suggestion for a future refactor - we really need to break up this test class. It was already pretty robust.

Reply:
@sfsholden Should I break it up into two files, one to test the default behavior as it exists today and one to test the behavior when the mobile feature flag is enabled?  When the feature flag is removed we could then simply remove one of the files.  The new file would probably still be around 1000 loc; should that be further broken up?

Reply:
@brandonpage I think this is fine for now. Maybe we could revisit the breakdown of this test class into more manageable pieces after the feature flag is removed.